Modules/CodeTrek/Config/config.php

```diff
@@ -33,5 +33,9 @@
             'label' =>'Onboarded',
             'slug'  =>'onboarded'
         ],
+        'preparatory' => [
+            'label' => 'Preparatory',
+            'slug' => 'preparatory'
+        ],
```

Comment:
This round is being shown at last. This round should be the first show in the round list.

```suggestion
        'preparatory' => [
            'label' => 'Preparatory',
            'slug' => 'Preparatory'
        ],
```